app/src/endpoints.c

```diff
@@ -118,6 +132,27 @@ int zmk_endpoints_toggle_transport(void) {
 
 struct zmk_endpoint_instance zmk_endpoints_selected(void) { return current_instance; }
 
+static struct zmk_endpoint_instance endpoints_add_extra_data(enum zmk_transport transport) {
```

Comment:
"add extra data" makes me think that the function would take a struct as an input and populate some fields on it, but this generates the entire struct. Maybe a name more like "get instance from transport" would be clearer?

Reply:
Name has been changed to `get_isntance_from_transport()`